data/building-hours.json

```diff
@@ -72,6 +120,35 @@
         }
     },
     {
+        "name": "Bookstore",
+        "image": "bookstore",
+        "times": {
+            "hours": {
+                "Mon": ["8:00", "17:00"],
+                "Tue": ["8:00", "17:00"],
+                "Wed": ["8:00", "17:00"],
+                "Thu": ["8:00", "17:00"],
+                "Fri": ["8:00", "17:00"],
+                "Sat": ["10:00", "16:00"]
+            }
+        }
+    },
+    {
+        "name": "Convenience Store",
+        "image": "convenience",
+        "times": {
+            "hours": {
+                "Mon": ["8:00", "20:00"],
+                "Wed": ["8:00", "20:00"],
```

Comment:
Not that the logic matters, but while we are being picky, would you mind flipping the order so that `Wed` and `Tue` switch spots, please?